### PR TITLE
Add feature content.

### PR DESCRIPTION
**Pull request by KonovalovaVV**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -62,6 +62,66 @@ <h1 class="title">
       </div>
     </div>
 
+    <div class="label">
```
Comment (MaksHladki) [inline index.html:65]: The `label` is not a meaningful name in this case.

File: css/main.css [hunk 2]
```diff
@@ -64,4 +64,57 @@
...
+
+.featured-section__content-item {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin:auto;
+    padding: 30px;
+}
+
+.featured-section__content-item--border {
```
Comment (MaksHladki) [inline css/main.css:100]: Please, get rid of this class. You can use `last-of-type` or `last-child`
https://developer.mozilla.org/ru/docs/Web/CSS/:last-of-type

File: index.html (hunk 1 above)
Comment (MaksHladki) [inline index.html:65]: Please, combine this section with `featured-section` into one.
Use the `section` HTML element for this new item.
https://developer.mozilla.org/ru/docs/Web/HTML/Element/section

File: index.html [hunk 3]
```diff
@@ -62,6 +62,66 @@ <h1 class="title">
...
+        <h2 class="header-label header-label--dark">Cras ornare tristique elit</h2>
+        <p class="header-descripton header-descripton--dark">Donec odio uisque volutpat mattis eros ullam malesuada erat
+          ut turpis. </p>
+      </div>
+    </div>
+
+    <div class="featured-section">
+      <div class="featured-section__content">
+
+        <div class="featured-section__content-item featured-section__content-item--border">
```
Comment (MaksHladki) [inline index.html:76]: Please, use another name. I guess this item can be reusable. What do you think of `featured-card`? And also mark it as `section` or `article`.

File: index.html [hunk 4]
```diff
@@ -62,6 +62,66 @@ <h1 class="title">
...
+          <i class="featured-section__content-icon fas fa-desktop"></i>
+          <div class="featured-section__content-lead lead">Nunc dignissim</div>
```
Comment (MaksHladki) [inline index.html:78]: I think it should be a title (h3). Please, update all other cards.

File: index.html [hunk 5]
```diff
@@ -62,6 +62,66 @@ <h1 class="title">
...
+            sagittis vel, euismod vel, velit.
+            Pellentesque egestas sem.
+          </div>
+          <button class="featured-section__content-button button">More</button>
+        </div>
+
+        <div class="featured-section__content-item">
+          <i class="featured-section__content-icon fas fa-bullhorn"></i>
+          <div class="featured-section__content-lead lead">Civamus vesti</div>
+          <div class="featured-section__content-description description">
```
Comment (MaksHladki) [inline index.html:113]: Please, use `p` instead of `div`.

File: css/main.css [hunk 6]
```diff
@@ -64,4 +64,57 @@
     margin: auto;
     color: #ffffff;
     font-size: 32px;
+}
+
+.label {
+    width: 100%;
+    height: 140px;
```
Comment (MaksHladki) [inline css/main.css:71]: Please, use paddings inst3ead of height. A user might have an opportunity to change the text.

File: index.html (hunk 1 above)
Comment (KonovalovaVV) [reply index.html:65]: Changed to header-section.

File: index.html (hunk 5 above)
Comment (KonovalovaVV) [reply index.html:113]: Ok.

Comment (KonovalovaVV) [reply index.html:113]: Ok.

File: css/main.css (hunk 6 above)
Comment (KonovalovaVV) [reply css/main.css:71]: Ok.

File: index.html (hunk 4 above)
Comment (KonovalovaVV) [reply index.html:78]: Ok.

File: index.html (hunk 1 above)
Comment (KonovalovaVV) [reply index.html:65]: Ok.

File: index.html (hunk 3 above)
Comment (KonovalovaVV) [reply index.html:76]: Ok.

File: css/main.css [hunk 7]
```diff
@@ -64,4 +64,57 @@
...
+.feature-card {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin:auto;
+    padding: 40px 30px;
+    border-right: 1px solid #b5c7c9;
+}
+
+section.feature-card:last-of-type {
```
Comment (MaksHladki) [inline css/main.css:100]: Please, avoid a combination of HTML tags and classes. BEM can easily resolve such situations.

File: index.html [hunk 8]
```diff
@@ -62,6 +62,69 @@ <h1 class="title">
...
+            erat
+            ut turpis. </p>
+        </div>
+      </div>
+
+      <div class="featured-section">
+        <div class="featured-section__content">
+
+          <section class="feature-card">
+            <i class="feature-card-icon fas fa-desktop"></i>
```
Comment (MaksHladki) [inline index.html:80]: I think it should be
feature-card__icon
feature-card__lead
feature-card__description
feature-card__button

Comment (KonovalovaVV) [reply index.html:80]: Ok, replaced.

**Pull request merged**